tasks-reference/pam/text.xml

```diff
@@ -99,16 +91,6 @@ The statement is composed of 3 or 4 tokens:
     passed to the module. These are module-dependent.
   </li>
 </ul>
-
-<p>
-As the number and the type of modules shipped with the implementation depends on
-the implementations themselves (Linux-PAM provides a full working set of
-modules, OpenPAM doesn't provide modules at all, and it's the operating system
-which provides them, as FreeBSD or NetBSD do), there are just a few modules
-which can be used directly in <c>pamd</c> files without the risk of providing a
-non-working configuration file:
-</p>
-
```

Comment:
You are removing the paragraph leading to a list. Should this now be merged with the `<ul/>` above instead?